api/src/shipit_api/admin/settings.py

```diff
@@ -99,7 +99,8 @@
 AUTH0_AUTH_SCOPES = assign_ldap_groups_to_scopes()
 
 # other scopes
-AUTH0_AUTH_SCOPES.update({"rebuild_product_details": LDAP_GROUPS["firefox-signoff"], "update_release_status": []})
+AUTH0_AUTH_SCOPES.update({"rebuild_product_details": LDAP_GROUPS["firefox-signoff"], "update_release_status": [], "create_product_channel_version/firefox": []})
```

Comment:
Another follow-up: We will need to grant releng scopes to seed the initial Thunderbird version once the Thunderbird bits are implemented.

```suggestion
AUTH0_AUTH_SCOPES.update(
    {
        "rebuild_product_details": LDAP_GROUPS["firefox-signoff"],
        "update_release_status": [],
        "create_product_channel_version/firefox": [],
        "create_product_channel_version/thunderbird": [],
    }
)
```